### ISSUE TITLE
ipecho plugin

### ISSUE THREAD

**Issue by Eun** [coredns/coredns#1122]
Howdy, 

I have written a simple plugin for coredns, I don't know if there is anyone (besides me) that can make use of it.  
But anyway I would be honored if you add it to your website.

[ipecho](https://github.com/Eun/coredns-ipecho) is a plugin that parses the IP out of a subdomain and echos it back as an record.
An simple example would be:
```
A IN 127.0.0.1.example.com. -> A: 127.0.0.1
```

Best regards

Eun

**Comment by miekg** [comment]
Nice :-) What *is* it's use?

You can open a PR on coredns.io to add it as an external plugin: https://coredns.io/2017/07/23/add-external-plugins/

**Comment by Eun** [comment]
I have a test domain that is used for local web development. It comes with an Let's Encrypt ssl certificate. (It is also here on [github](https://github.com/Eun/test.bi)).  
However I noticed that Let's Encrypt wants to issue wildcard certificates as well (in 2018), so I tought it would be a good idea to skip the DNS reroute drama. (Hosts file, etc...).  
This means you have a domain with an valid ssl certificate that can be used localy.

Thats why I created this plugin.
However, I am thinking whether I should create another plugin for letsencrypt dns challenges. mmh...
